conanfile.py

```diff
@@ -41,13 +41,13 @@ class UserverConan(conans.ConanFile):
     default_options = {
         'shared': False,
         'fPIC': True,
-        'with_jemalloc': False,
+        'with_jemalloc': True,
         'with_mongodb': True,
         'with_postgresql': True,
         'with_postgresql_extra': False,
         'with_redis': True,
-        'with_grpc': False,
-        'with_clickhouse': False,
+        'with_grpc': True,
+        'with_clickhouse': True,
```

Comment:
Please disable clickhouse by default due to there is no conan package for clickhouse or just mention this if you think that it's a very important module